nocodes/src/main/java/io/qonversion/nocodes/internal/di/network/NetworkAssemblyImpl.kt

```diff
@@ -24,7 +24,7 @@ internal class NetworkAssemblyImpl(
 
     override fun networkClient(): NetworkClient = NetworkClientImpl(
         miscAssembly.jsonSerializer(),
-        isFallbackAvailable = internalConfig.primaryConfig.fallbackFileName != null
+        isFallbackAvailable = internalConfig.primaryConfig.customFallbackFileName != null
```

Comment:
We can't assume that fallback file is unavailable if there is no custom file name. It still can exist with the default name.